File: Python/SistemaLibreria/Usuarios.py
import random
class Usuarios:
    __id = 0
    __name = ""
    __age = 0
    __librosRentados = []

    def __init__(self, name, age, id ):
        self.__id = random.randint(1,1000)
        self.__name = name
        self.__age = age
        self.__librosRentados = []
    
    def getId(self):
        return self.__id
    
    def setId(self, id):
        self.__id =id
    
    def getName(self):
        return self.__name
    
    def setName(self, name):
        self.__name = name
    
    def getAge(self):
        return self.__age
    
    def setAge(self, age):
        self.__age =age
    
    def getLibrosRentados(self):
        return self.__librosRentados
    
    def rentarLibro(self, nuevoLibro):
         self.__librosRentados.append(nuevoLibro)

    

File: Java/SistemaLibreria/src/sistemalibreria/Usuarios.java

package sistemalibreria;
import java.util.*;


public class Usuarios 
{
    Scanner leer = new Scanner(System.in);
    Random aleatorio = new Random();
    
    private int id;
    private String name;
    private int age;
    private List<Libro> librosRentados;
    
    public Usuarios( String name, int age, int id)
    {
        this.id = aleatorio.nextInt(1000)+1;
        this.name = name;
        this.age = age;
         this.librosRentados = new ArrayList<>();
    }

    public Scanner getLeer() {
        return leer;
    }

    public void setLeer(Scanner leer) {
        this.leer = leer;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }
    
   public List<Libro> getLibrosRentados() {
        return librosRentados;
    }

    public void rentarLibro(Libro libro) {
        librosRentados.add(libro);
    }
 
}
